.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+<!--
+Please make sure you've read and understood our contributing guidelines;
+https://github.com/checkpoint-restore/criu/blob/criu-dev/CONTRIBUTING.md
+
+In short you need to:
+
+- Describe What you do and How you do it;
+- Separate each logical change into a separate commit;
+- Add a "Signed-off-by:" line identifying that you certify your work with DCO;
+- If you fix some specific bug or commit, please add "Fixes: ..." line;
+- Review fixes should be merged to original commits;
```

Comment:
It might be good to clarify here about the use of `git rebase` or `git commit --amend`, as opposite to `git merge`.

Reply:
I wannna be short here, but I will try.

Reply:
I wanna be short here, but I would try.